lib/Models/Catalog/Ows/WebProcessingServiceCatalogFunction.ts

```diff
@@ -693,6 +706,74 @@ const GeoJsonGeometryConverter = {
   }
 };
 
+const FileConverter = {
+  inputToParameter: function (
+    catalogFunction: CatalogFunctionMixin.Instance,
+    input: Input,
+    options: FunctionParameterOptions
+  ) {
+    const formats = input.ComplexData?.Supported?.Format
+      ? Array.isArray(input.ComplexData.Supported.Format)
+        ? input.ComplexData.Supported.Format
+        : [input.ComplexData.Supported.Format]
+      : undefined;
+
+    const supportedFormats = formats
+      ? filterOutUndefined(
+          formats.map((format) => {
+            if (format?.MimeType) {
+              if (
+                format.Encoding === undefined ||
+                format.Encoding === "base64"
+              ) {
+                return {
+                  mimeType: format.MimeType,
+                  encoding: format.Encoding ?? "none"
+                } as SupportedFormat;
```

Comment:
Can the cast be avoided by specifying the return type of the `(format) => ` function?